src/api/posts/src/routes/feeds.js

```diff
@@ -34,6 +34,32 @@ feeds.get('/', async (req, res, next) => {
   );
 });
 
+//This should return a JSON array of Objects with the feed info as well as the reason why it was invalidated.
+feeds.get('/invalid', async (req, res, next) => {
+  let keys = [];
+  try {
+    keys = await getInvalidFeeds();
+  } catch (error) {
+    logger.error({ error }, 'Unable to get invalid feeds from Redis');
+    next(error);
```

Comment:
You need a `return` here so that it doesn't keep going.

Reply:
Thank you Prof, I will fix it